packages/federatorai/federatorai.patch

```diff
@@ -0,0 +1,18 @@
+diff -x '*.tgz' -x '*.lock' -uNr packages/federatorai/charts-original/Chart.yaml packages/federatorai/charts/Chart.yaml
+--- packages/federatorai/charts-original/Chart.yaml
++++ packages/federatorai/charts/Chart.yaml
+@@ -1,5 +1,8 @@
+ apiVersion: v1
+ appVersion: 4.5.1-ga
++annotations:
++  catalog.cattle.io/certified: partner
++  catalog.cattle.io/release-name: federatorai
+ description: Federator.ai helps enterprises optimize cloud resources, maximize application performance, and save significant cost without excessive over-provisioning or under-provisioning of resources, meeting the service-level requirements of their applications.
+ home: https://www.prophetstor.com
+ icon: https://raw.githubusercontent.com/prophetstor-ai/public/master/images/logo.png
+@@ -16,4 +19,4 @@
+ name: federatorai
+ sources:
+ - https://www.prophetstor.com
+-version: 4.5.1
++version: 4.5.1-1
```

Comment:
We append the packageVersion to the chart's version to generate the version of the new package that gets generated. Since you are appending -1 here, the package's version will result in 4.5.1-1<packageVersion> instead of 4.5.1<packageVersion>-1 as we expect. Is this a necessary change, or is it possible this can be reverted?